scenarios/scenario1/variables.tf

```diff
@@ -1,11 +1,16 @@
 variable "region" { default = "eu-de" }
+variable "az_default" { default = false}
 variable "availability_zone" { default = "eu-de-03" }
+variable "availability_zones" {
+  type        = list(string)
+  default     = ["eu-de-01", "eu-de-02", "eu-de-03"]
+}
 variable "ecs_flavor" {}
 variable "ecs_image" {}
 variable "addr_3_octets" { default = "192.168.0" }
 variable "scenario" {}
 variable "nodes_count" {
   type    = number
-  default = 2
+  default = 3
```

Comment:
Let's remove default value here, it's not really used

Reply:
done